diff --git a/src/OMBManagerInstall.py b/src/OMBManagerInstall.py
index b13b237..4dc8409 100644
--- a/src/OMBManagerInstall.py
+++ b/src/OMBManagerInstall.py
@@ -343,7 +343,22 @@ class OMBManagerInstall(Screen):
 		os.system(OMB_UMOUNT_BIN + ' ' + ubi_path)
 		os.system(OMB_UBIDETACH_BIN + ' -m ' + mtd)
 		os.system(OMB_RMMOD_BIN + ' nandsim')
-
+		
+		if not os.path.exists('/usr/lib/python2.7/boxbranding.so'):
+			os.system("ln -s /usr/lib/enigma2/python/boxbranding.so /usr/lib/python2.7/boxbranding.so")
+		if os.path.exists(dst_path + '/usr/lib/python2.7/boxbranding.py'):
+			os.system("cp /usr/lib/enigma2/python/boxbranding.so " + dst_path + "/usr/lib/python2.7/boxbranding.so")
+			os.system("rm -f " + dst_path + '/usr/lib/python2.7/boxbranding.py')
+		if not os.path.exists(dst_path + "/usr/lib/python2.7/subprocess.pyo"):
+			os.system("cp /usr/lib/python2.7/subprocess.pyo " + dst_path + "/usr/lib/python2.7/subprocess.pyo")
+		if dst_path.find('OpenPLi') != -1:
+			import fileinput
+			for line in fileinput.input(dst_path + '/etc/init.d/volatile-media.sh', inplace=True):
+				if 'mount -t tmpfs -o size=64k tmpfs /media' in line:
+					print "mountpoint -q \"/media\" || mount -t tmpfs -o size=64k tmpfs /media"
+				else:
+					print line.rstrip()
+		
 		return True
 
 	# Based on nfi Extract by gutemine
